sdk/identity/azure-identity/src/main/java/com/azure/identity/credential/AuthorizationCodeCredentialBuilder.java

```diff
@@ -39,11 +39,11 @@ public AuthorizationCodeCredentialBuilder authorizationCode(String authCode) {
      * to complete the {@link AuthorizationCodeCredential} construction from there.
      * This is also called Reply URLs in some contexts.
      *
-     * @param redirectUri the redirect URI to send the authorization code
+     * @param redirectUrl the redirect URL to send the authorization code
```

Comment:
The issue with this change is that this does not match the name in other places / libraries Microsoft publishes e.g. https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/dev/src/main/java/com/microsoft/aad/msal4j/AuthorizationCodeParameters.java#L41.

@JonathanGiles

Reply:
From the reviews we did 'offline', I still lean towards URL.